pkgs/by-name/wi/wivrn/package.nix

```diff
@@ -0,0 +1,112 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchFromGitLab
+, avahi
+, boost
+, cmake
+, cudaPackages
+, eigen
+, ffmpeg
+, freetype
+, git
+, glm
+, glslang
+, harfbuzz
+, libdrm
+, libGL
+, libva
+, libpulseaudio
+, libX11
+, libXrandr
+, nlohmann_json
+, onnxruntime
+, openxr-loader
+, pkg-config
+, python3
+, shaderc
+, spdlog
+, systemd
+, udev
+, vulkan-headers
+, vulkan-loader
+, vulkan-tools
+, x264
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "wivrn";
+  version = "0.14.1";
+
+  src = fetchFromGitHub {
+    owner = "meumeu";
+    repo = "wivrn";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-dnc9UNETDzT+sqo9bSTP1qZs/7kWftDo50yRgP94Mh4=";
+  };
+
+  monadoSrc = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = "monado";
+    repo = "monado";
+    # Version stated in CMakeList for WiVRn 0.14.1
+    rev = "ffb71af26f8349952f5f820c268ee4774613e200";
+    hash = "sha256-+RTHS9ShicuzhiAVAXf38V6k4SVr+Bc2xUjpRWZoB0c=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    cudaPackages.cuda_nvcc
+    git
+    pkg-config
+    python3
+  ];
+
+  buildInputs = [
+    avahi
+    boost
+    cudaPackages.cuda_cudart
+    eigen
+    ffmpeg
+    freetype
+    glm
+    glslang
+    harfbuzz
+    libdrm
+    libGL
+    libva
+    libX11
+    libXrandr
+    libpulseaudio
+    nlohmann_json
+    onnxruntime
+    openxr-loader
+    shaderc
+    spdlog
+    systemd
+    udev
+    vulkan-headers
+    vulkan-loader
+    vulkan-tools
+    x264
+  ];
+
+  cmakeFlags = [
+    (lib.cmakeBool "WIVRN_BUILD_CLIENT" false)
```

Comment:
I'd would like it if the client build was enabled and put in a seperate output, but I wouldn't consider this a blocker.

Reply:
Indeed, not a blocker. The PC test client build seems to be an internal artifact and is absolutely not relevant for the server part, and is therefore out of scope for this PR.